db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171207164824) do
+ActiveRecord::Schema.define(version: 20171211164321) do
```

Comment:
Do we need this version update now that this migration is being inserted back in time?